oshi-demo/src/main/java/oshi/demo/OshiGui.java

```diff
@@ -85,6 +72,7 @@ private void init() {
         new Thread(new AddMenuBarTask("FileStores", 'F', "FileStore Usage", new FileStorePanel(si))).start();
         new Thread(new AddMenuBarTask("Processes", 'P', "Processes", new ProcessPanel(si))).start();
         new Thread(new AddMenuBarTask("USB Devices", 'U', "USB Device list", new UsbPanel(si))).start();
+        new Thread(new AddMenuBarTask("Interfaces", 'I', "Interface list", new InterfacePanel(si))).start();
```

Comment:
Thanks for adding to the demo! I'm wondering if "Network Interfaces" would be a better title (with `N` mnemonic) although I don't have a strong opinion.

Reply:
We could also put some of the "Network Params" results in the top of a "Network" pane.